manywheel/build_rocm.sh

```diff
@@ -114,7 +114,11 @@ elif [[ "$OS_NAME" == *"Ubuntu"* ]]; then
     LIBGOMP_PATH="/usr/lib/x86_64-linux-gnu/libgomp.so.1"
     LIBNUMA_PATH="/usr/lib/x86_64-linux-gnu/libnuma.so.1"
     LIBELF_PATH="/usr/lib/x86_64-linux-gnu/libelf.so.1"
-    LIBTINFO_PATH="/lib/x86_64-linux-gnu/libtinfo.so.5"
+    if [[ $ROCM_INT -ge 50300 ]]; then
+        LIBTINFO_PATH="/lib/x86_64-linux-gnu/libtinfo.so.6"
+    else
+        LIBTINFO_PATH="/lib/x86_64-linux-gnu/libtinfo.so.5"
```

Comment:
Side-note on ROCm5.2 (and earlier): We need the version-dependent switch because ROCm5.2 needs libtinfo.so.5, otherwise we get the following message when using libtinfo.so.6 for it:
/usr/bin/ld: /tmp/libtorch/lib/libamd_comgr.so: undefined reference to 'setupterm@NCURSES_TINFO_5.0.19991023'